zerver/webhooks/jotform/doc.md

```diff
@@ -0,0 +1,16 @@
+Get Zulip notifications for your JotForm responses!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+   By default, the integration will use the name of ***JotForm-Response*** as topic. Make sure to ***not use colon*** as part of form's questions.
```

Comment:
Can you explain for me what problem this is meant to avoid?

Reply:
I have picked up the responses from the ***pretty*** key of the payload in which the fields are separated by commas and the key value pair separated by colon. So to pick up the fields I have split them based on the appearance of colon and another colon in the question will interfere in this process when I performed the test. Hence I have added this to the readme.